hepcrawl/testlib/fixtures.py

```diff
@@ -10,26 +10,31 @@
 from __future__ import absolute_import, division, print_function
 
 import os
+import json
 
 from scrapy.http import Request, TextResponse
 from scrapy.selector import Selector
 
 
-def fake_response_from_file(file_name, url='http://www.example.com', response_type=TextResponse):
+def fake_response_from_file(file_name, test_suite='unit', url='http://www.example.com', response_type=TextResponse):
     """Create a Scrapy fake HTTP response from a HTML file
 
-    :param file_name: The relative filename from the responses directory,
-                      but absolute paths are also accepted.
-    :param url: The URL of the response.
-    :param response_type: The type of the scrapy Response to be returned,
-                          depending on the Request (Response, TextResponse, etc)
-
-    :returns: A scrapy HTTP response which can be used for unittesting.
+    Args:
+        file_name: The relative filename from the responses directory,
+            but absolute paths are also accepted.
+        test_suite: The test suite that the response file comes from,
+            e.g. ``unit``, ``functional``.
+        url: The URL of the response.
+        response_type: The type of the scrapy Response to be returned,
+            depending on the Request (Response, TextResponse, etc).
+
+    Returns:
+        A scrapy HTTP response which can be used for unit testing.
```

Comment:
Here the type of the response is `response_type`, you can also rephrase to say that it will use the response_type param as the class for it.